src/Command.php

```diff
@@ -85,4 +85,12 @@ public function getArtisanCommand(): ArtisanCommand
     {
         return $this->artisanCommand;
     }
+
+    public function shouldOpenArgumentsAccordionOnLoad(): bool
+    {
+        if (!$this->hasArguments())
+            return false;
+
+        return $this->getArguments()->filter(fn(CommandArgument $argument) => $argument->isRequired())->count() > 0;
```

Comment:
Since `getArguments()` always return a collection, we can make this function quite a bit shorter. I also like the use of `contains` for this purpose. 🙂 
```suggestion
        return $this->getArguments()->contains(fn(CommandArgument $argument) => $argument->isRequired());
```